java/src/main/java/ai/rapids/cudf/Scalar.java

```diff
@@ -140,15 +140,15 @@ public static Scalar fromUnsignedShort(Short value) {
     return Scalar.fromUnsignedShort(value.shortValue());
   }
 
-  public static Scalar fromDurationDay(int value) {
+  public static Scalar durationDayFromInt(int value) {
```

Comment:
Should be plural just as it is for seconds, milliseconds, etc.  and consistent with the naming of the corresponding methods in `ColumnVector` and `HostColumnVector`.
```suggestion
  public static Scalar durationDaysFromInt(int value) {
```

Reply:
These new methods are also missing Javadoc.

Reply:
I will add javadocs for this method but most of the public methods in this class don't have javadocs may be we should create a follow-on issue to address all of them?

Reply:
Yes, we can address the missing Javadoc for the others in a followup PR.